src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.33.1'
+__version__ = '1.34.1'
```

Comment:
Interesting version bump...

Reply:
I'm left guessing what you mean.  As per semver.org:

Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards compatible manner, and
    PATCH version when you make backwards compatible bug fixes.

It's new functionality that is backwards compatible, hence a minor version bump.